lib/containers/artist.js

```diff
@@ -63,6 +63,7 @@ export default Relay.createContainer(Artist, {
       fragment on Artist {
         ${Header.getFragment('artist')}
         ${Biography.getFragment('artist')}
+        ${Articles.getFragment('artist')}
```

Comment:
If you’re changing to the `@relay(plural: true)` syntax, then this would become something like:

```
articles {
  ${Articles.getFragment('articles')}
}
```